### DISCUSSION TITLE
Usage of "once a day" model instead of "once every 24 hours" model

### DISCUSSION THREAD

**Discussion by TheAshenWolf** [ServerListPlus/issues#14] [Ideas]
24 hour model means you have to vote every day in the same second, to ensure you can vote every day endlessly... lets be honest, that's not possible.

Daily model ensures you vote once a day... essentially making the same thing, but it does not mater when, during the said day, you vote... two votes would have a time difference between 1 minute and 47:59 hours, but in the end, the average would be once again, 24 hours

This way it would allow people to vote more flexibly during the day.

**Comment by Brend-Smits** [comment]
Thanks, this makes sense to implement. I will create an issue in our backend and add it to the roadmap

**Reply by MaxKorlaar** [reply]
Great suggestion, this makes way more sense. Maybe we should also clarify which timezone we're using for that then.

**Reply by Brend-Smits** [reply]
We could also show a countdown on however or something to when it's the 'next' day.

**Comment by Brend-Smits** [comment]
I've added this to the roadmap: https://github.com/ServerListPlus/issues/issues/24
I will look into implementing this asap.

**Reply by Brend-Smits** [reply]
Implementation is finished. This will be included in the next release :)
